src/Client/BatchRequestFailureException.php

```diff
@@ -16,20 +15,17 @@ class BatchRequestFailureException extends Exception
      */
     private $batchRequest;
     /**
-     * @var BatchResults
+     * @var array
      */
     private $batchResults;
 
     /**
      * @param BatchRequest $batchRequest
-     * @param BatchResults $batchResults
+     * @param array $batchResults
      */
-    public function __construct(BatchRequest $batchRequest, BatchResults $batchResults)
+    public function __construct(BatchRequest $batchRequest, array $batchResults)
```

Comment:
Maybe it is an array of some meaningful objects? Or it is just array of anything? We can maybe put here `ObjectList` ?